package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "chablis",
-  "version": "1.0.11",
+  "name": "chablis1",
```

Comment:
I agree for `chablis`
agree to version correct

Reply:
Oh, excelent comment. I've fixed this on this commit - e9686b1c98662e2f6e77efb3d11883eeeb52b56c